_examples/sql/event-log/README.md

```diff
@@ -0,0 +1,51 @@
+# Event log (Kafka to MySQL)
+
+This example shows how to use the SQL Publisher from [SQL Pub/Sub](https://github.com/ThreeDotsLabs/watermill-sql).
+
+## Background
+
+Some PubSubs (e.g. Kafka) come with support for storing processed messages, possibly even with no expiration date.
+This can be useful for audit purposes or to reply selected messages again in the future. But what if you'd like to use
+a PubSub that offers no storage and an event log is needed?
+
+## Solution
+
+Plugging a pair of a publisher and a subscriber into Watermill's `Router` can work as a proxy from one PubSub to another.
+To ensure that events are written to a persistent storage, you can use the SQL publisher.
+
+Google Cloud Pub/Sub subscriber consumes events from a topic and inserts them into a MySQL table. While this particular 
+PubSub doesn't guarantee proper order of messages, you can use `OccurredAt` field of the payload for sorting.
+Google Cloud Pub/Sub is used just as an example and any other subscriber can be used instead.
+
+The example uses `DefaultSchema`, but you can define your own table definition and queries.
+See [SQL Pub/Sub documentation](https://watermill.io/pubsub/sql) for details.
+
+## Requirements
+
+To run this example you will need Docker and docker-compose installed. See installation guide at https://docs.docker.com/compose/install/
+
+## Running
+
+```bash
+docker-compose up
+```
+
+After few seconds, some events should be saved in the table:
+
+```
+docker-compose exec mysql mysql -e 'select * from watermill.watermill_events;'
++--------+--------------------------------------+---------------------+---------------------+----------+
+| offset | uuid                                 | created_at          | payload             | metadata |
++--------+--------------------------------------+---------------------+---------------------+----------+
+|      1 | 2faf6a14-f52a-4d6c-a4be-7355db428be1 | 2019-08-17 12:23:35 | {"num": 1566044614} | {}       |
```

Comment:
the payload in this example has `Name` and `OccuredAt`, while this table has `"num"`
Did I miss something or is it a bad copypaste?

Reply:
I've updated the code but not the README. However now the table takes too much space to show it nicely, so I've just cut the payload: be799a0ab8e0d60b3cb5177b8fe3f03c70316900

Reply:
:+1: